src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/PrisonAppointment.kt

```diff
@@ -43,6 +43,8 @@ class PrisonAppointment private constructor(
   val startTime: LocalTime,
 
   val endTime: LocalTime,
+
+  var notesForPrisoners: String? = null,
 ) {
```

Comment:
Leaving both `comments` and `notesForPrisoners` in initially?

Reply:
Yes this is intentional.